eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -49,6 +46,19 @@ class ReindexCommand extends ContainerAwareCommand
     /** @var string */
     private $projectDir;
 
+    /** @var \eZ\Publish\SPI\Search\Content\IndexerGateway */
+    private $gateway;
+    /** @var \eZ\Publish\SPI\Persistence\Content\Location\Handler */
+    private $locationHandler;
```

Comment:
```suggestion

    /** @var \eZ\Publish\SPI\Persistence\Content\Location\Handler */
    private $locationHandler;
```

Nitpick: missing new line